src/test/resources/templates/V103__Trigger_template_duplicate_host_g_x_log_g.sql

```diff
@@ -0,0 +1,78 @@
+use cfe_18;
+/*
+    Template for building trigger on host_groups_x_capture_groups
+    1.  Gather hosts from the host_group which is being added.
+    2.  Gather all the host_groups which those hosts are included in.
+    3.  Gather all the capture_groups which are linked to host_groups mentioned before.
+    4.  Gather all the tags included in capture_groups which we collected.
+    5.  Creates unique check on tags. On collision trigger will stop the host_group and capture_group linkage.
+ */
+
+#### Before insert into host_groups_x_capture_groups
+
+-- napataan kaikki hostit mitkä tulee uudesta ryhmästä
```

Comment:
use english and declare these as examples of the idea behind g_x_g triggers

Reply:
Fixed